Docs/source/Hardware/Boards.rst

```diff
@@ -427,15 +541,593 @@ the +5V input and corresponding grounds are already isolated from the +12V line.
-and the mux switches between all of them.
+Overview
+^^^^^^^^
+`GitHub Link <https://github.com/lhr-solar/BPS-MinionPCB>`__
+
+`BOM Link <https://www.mouser.com/ProjectManager/ProjectDetail.aspx?AccessID=14BB50AAA1>`__ 
+
+
+Brief Description/Purpose: 
+    There are two temperature sensors for each battery module with a module minion board for each of the 
+    four rows. Each board measures up to 12 battery modules and 16 temperature sensors, but is configured 
+    for eight modules and 16 sensors by default. The temperature sensors are placed as inputs to a mux 
+    and the mux switches between all of them.
 
 .. figure:: ../_static/LTC6811.png
     :align: center
 
     LTC6811 Block Diagram 
+
+Pertinent Regulations
+^^^^^^^^^^^^^^^^^^^^^
+===========  ============================================== ===========================================================
+Regulation   Description of Regulation                      How Regulation is Met
+
+**8.3.A.5**  | System in which measurements are constantly  | The Minion board constantly takes voltage and temperature 
+             | monitored and where actions are taken        | measurements of the battery modules and transmits data
+             | immediately without operator intervention to | back to the leader board.
+             | open the Main Power Switch should a battery  
+             | Protection Fault occur. 
+===========  ============================================== ===========================================================
+
+Context
+^^^^^^^
+**Location of the Board:** Battery box 
+
+**List of I/O and Connections:**
+    
+    * Leaderboard/Minion Board Input (Isolated SPI bus)
+        * Input from BPS Leader Board, tells the Minion board when to gather measurments form the batteries. 
+    * LeaderBoard/Minion Board Output (Isolated SPI bus)
+        * Outputs to the leaderboard, transmits voltage and temperature readings from the battery modules.
+    * Voltage Connectors 
+        * Input from the battery modules, transmit voltage from each battery module. The IC also uses this 
+          voltages to power itself. 
+    * Minion Shield Power  
+        * 5V output from the LTC6811 to the Minion Shield board. This is used to power the temperature sensors.
+    * Temperature Input
+        * Input from the Minion Shield board, transmits temperature data.
+
+Schematic
+^^^^^^^^^
+
+**Main**
+++++++++
+
+*What does this circuit do?*
+    This circuit measures the voltages and temperatures of each battery module 
+    and sends the data to the leader board when prompted. 
+*Why do we need it?*
+    Regulations stipulate that the voltages and temperatures of the battery 
+    modules must be monitored at all times. 
+*List of Circuit Components*
+    * 2 Molex Micro-fit 3.0 1x07 3.00mm Horizontal  
+        * Description: Each connector allows for eight connections to the battery modules 
```

Comment:
seven connections?

Reply:
not sure why the original doc says eight but I'm sure they meant seven